GVFS/GVFS.Service/Mac/CurrentUser.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GVFS.Common.Tracing;

```

Comment:
This is just a placeholder class for now. Its there because `NotificationHandler` class which is included in the project references it.